doc/sphinx/source/recipes/recipe_sea_surface_salinity.rst

```diff
@@ -202,8 +202,14 @@ References
 Example plots
 -------------
 
-.. figure:: /recipes/figures/sea_surface_salinity/radar.png
+.. figure:: /recipes/figures/sea_surface_salinity/radar_bias.png
    :align: center
 
-   Radar plot showing correlation of average sea surface salinity for multiple
+   Radar plot showing bias of average sea surface salinity for multiple
+   regions with the observations
+
+.. figure:: /recipes/figures/sea_surface_salinity/radar_std.png
+   :align: center
+
+   Radar plot showing standard deviation ratio of average sea surface salinity for multiple
```

Comment:
Change lines 214-215 for "Radar plot showing the ratio between the simulated and observed standard deviations of the regional averages of sea surface salinity in the selected ocean basins and seas"